tests/test_velocity_model_plots.py

```diff
@@ -8,6 +8,7 @@
 PLOT_IMAGE_DIRECTORY = Path("wiki/images")
 INPUT_FFP = Path(__file__).parent / "input"
 
+# Kept at: QuakeCoRE/Jake Faulkner/rho3dfile in Dropbox.
```

Comment:
Perhaps this should be in the QuakeCoRE Public folder on Dropbox?

Reply:
yup you're right. Will move it.